extensions/circuit/circuit.yml

```diff
@@ -69,6 +67,71 @@ nodes:
         optional: false
         cardinality: one
         kind: Attribute
+      - name: location
+        label: Location
+        peer: LocationHosting
+        optional: true
+        cardinality: one
+        kind: Attribute
+        order_weight: 1500
+      - name: enpoints
+        peer: DcimCircuitEndpoint
+        optional: true
+        cardinality: many
+        kind: Component
+
+  - name: CircuitEndpoint
+    namespace: Dcim
+    description: A circuit endpoint, could be a position in a MMR...
+    label: Circuit Endpoint
+    inherit_from:
+      - DcimEndpoint
+    icon: mdi:ethernet
+    menu_placement: DcimCircuit
+    human_friendly_id:
+      - circuit__circuit_id__value
+      - name__value
+    uniqueness_constraints:
+      - [circuit, name__value]
+    order_by:
+      - name__value
+    display_labels:
+      - name__value
+    attributes:
+      - name: name
+        kind: Text
+        description: Name of the circuit endoint, could be a MMR position for instance.
+        order_weight: 1000
```

Comment:
You have oder_weight only on the name, you may want it on status and description as well